<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterListForm</class>
 <widget class="QMainWindow" name="FilterListForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dictionary filters</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="1" column="0" colspan="4">
     <widget class="ZListView" name="filterTable">
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>185</width>
        <height>6</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QToolButton" name="addButton">
      <property name="text">
       <string>Add</string>
      </property>
      <property name="icon">
       <iconset resource="Resources/resources.qrc">
        <normaloff>:/addmark.svg</normaloff>:/addmark.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>18</width>
        <height>18</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="delButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
      <property name="icon">
       <iconset resource="Resources/resources.qrc">
        <normaloff>:/delmark.svg</normaloff>:/delmark.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>18</width>
        <height>18</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="editButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
      <property name="icon">
       <iconset resource="Resources/resources.qrc">
        <normaloff>:/penciledit.svg</normaloff>:/penciledit.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>18</width>
        <height>18</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="2" margin="4"/>
 <customwidgets>
  <customwidget>
   <class>ZListView</class>
   <extends>QTableView</extends>
   <header>zlistview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>addButton</tabstop>
  <tabstop>delButton</tabstop>
  <tabstop>editButton</tabstop>
  <tabstop>filterTable</tabstop>
 </tabstops>
 <resources>
  <include location="Resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
